5.6/atomic-contrib/include.sh

```diff
@@ -0,0 +1,12 @@
+# Check whether all required variables are set
```

Comment:
@hhorak cab you rename this file to "variables.sh" or "vars.sh" ? As this file really just define variables used by other scripts.